libs/datamodel/connectors/datamodel-connector/src/lib.rs

```diff
@@ -35,6 +35,12 @@ pub trait Connector: Send + Sync {
     /// This powers the use of native types for QE + ME.
     fn parse_native_type(&self, name: &str, args: Vec<String>) -> Result<NativeTypeInstance, ConnectorError>;
 
+    /// This function is used in ME for error messages
+    fn render_native_type(&self, native_type: serde_json::Value) -> String {
+        let instance = self.introspect_native_type(native_type).unwrap(); //todo
```

Comment:
is this likely to crash?

Reply:
Probably not, I think the native types json here should already have passed validation no matter where it is coming from
I'll remove the todo